File: d17Sat/C/program/p00check_id/checkIdMain.c
#include "checkId.c"
#include <stdlib.h>

int main(int argc, char *argv[]){
  struct CheckId ckd;
  // ckd.id = atoi(argv[1]);
  ckd.id = malloc(sizeof(char));
  ckd.pw = malloc(sizeof(char));
  puts("Enter your ID");
  scanf("%s", ckd.id);
  puts("Enter your PW");
  scanf("%s", ckd.pw);
  checkId(ckd);

  return 0;
}

File: d17Sat/Php/program/p00check_id/CheckIdMain.php
<?php
include "CheckId.php";

$id = readline("Enter your id:");
$pw = readline("Enter your password:");
$ckd = new CheckId($id, $pw);

/* $ckd->setPw("1234");
echo $ckd->getPw(); */

$ckd->setPw("113");

$ckd->checkId();


